### PR TITLE
add new framework django and the django-appwrite package description …

### PR DESCRIPTION
**Pull request by khashashin**
[link to source code](https://github.com/khashashin/django-appwrite)

## What does this PR do?

This Pull Request (PR) adds a new category "django" and new information about the "django-appwrite" package. The new category "django" will provide a specific area for information related to the Django web framework, and the "django-appwrite" package information will give details about a package that provides integration between Django and the Appwrite Server.

The purpose of adding this information is likely to make it easier for users to find information related to Django and the integration between Django and the Appwrite Server. By having a separate category for Django and providing detailed information about the "django-appwrite" package, it will make it easier for users to understand the purpose and use of the package and how it can be used with Django.

This PR is likely to improve the overall organization and usability of the information related to Django and the Appwrite Server for users who want to integrate the two technologies.

## Test Plan

Nothing was changed except the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,6 +283,9 @@ Share your apps here! Submit a pull request!
 
 * **Discordly** A Discord-bot to readily and easily shorten links from Discord itself, made possible by Appwrite. [Source Code](https://github.com/Arpan-206/discordly)
 
+### Django
+* **Authenticate Appwrite User in Django** - A Django middleware to authenticate users with Appwrite. [Source Code](https://github.com/khashashin/django-appwrite)
```
Comment (adityaoberai) [inline README.md:287]: Could we shift this to the Python category?

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Hey @khashashin 
Thank you so much for your contribution. We definitely appreciate your work on the package you've created.

Would it be alright if, for the time-being, we could shift this to the Python category? The reason I say so is because the Python category itself has some way to grow in our repository. Once it matures and warrants a framework-based breakup, we can then categorise it in a separate Django category.

Please let me know if that sounds alright.

**Comment by khashashin** [comment]
Hi @adityaoberai sure! Done! :)

**Review by adityaoberai — APPROVED** [review]
Thank you so much for your contribution, @khashashin! It looks great and I'm happy to go ahead and merge it in the repo 👍

**Pull request merged**